src/Aspire.Hosting.Azure/AzureStorageEmulatorResource.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.Azure;
+
+public class AzureStorageEmulatorResource(AzureStorageResource innerResource) : ContainerResource(innerResource.Name), IResource
```

Comment:
All public APIs should have triple slash comments. We should not merge any PRs without them, perhaps we can fail the build or something if they're missing? //cc @davidfowl and @mitchdenny

Reply:
Yes, we should start to do that.

Reply:
cc @joperezr